lib/index.js

```diff
@@ -122,7 +122,9 @@ function convertDates(properties) {
 function formatTraits(traits) {
   var ret = {};
   each(function(value, key) {
-    var k = key.toLowerCase().replace(/\s/g, '_');
+    // Using split/join due to IE 11 failing to properly support regex in str.replace()
+    // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/@@replace
+    var k = key.toLowerCase().split(' ').join('_').split('.').join('_');
```

Comment:
Are all this cases covered here? (or do we need to cover all of them?)
* A space character
* A tab character
* A carriage return character
* A new line character
* A vertical tab character
* A form feed character